eng/Packaging.props

```diff
@@ -12,6 +12,9 @@
     <PackagePlatform Condition="'$(PackagePlatform)' == 'amd64'">x64</PackagePlatform>
     <NativePackagePath>$(MSBuildThisFileDirectory)src/Native/pkg</NativePackagePath>
 
+    <!-- Used by PackageLibs.targets -->
+    <XmlDocFileRoot>$(NuGetPackageRoot)$(MicrosoftPrivateIntellisensePackageId)/$(MicrosoftPrivateIntellisensePackageVersion)/xmldocs/netcoreapp</XmlDocFileRoot>
```

Comment:
Just confirming, do we no longer need `XmlDocDir` (from https://github.com/dotnet/corefx/pull/38151/files#diff-ba98fbb097dbdca16daf37fed8c21357L18)?

Reply:
XmlDocDir is still defined, just in a different place: https://github.com/dotnet/corefx/pull/38151/files#diff-8b8f08ffbf7b863fb3700c1718eeb4cbR20